docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -456,6 +456,42 @@ SELECT *
 FROM example.testdb.customer_orders FOR VERSION AS OF 3
 ```
 
+A different approach of retrieving historical data is to specify a point in time
+in the past, such as a day or week ago. The latest snapshot of the table taken
+before or at the specified timestamp in the query is internally used for
+providing the previous state of the table:
+
+```sql
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna';
+```
+
+The connector allows to create a new snapshot through DeltaLake's [replace table](delta-lake-create-or-replace).
+
+```sql
+CREATE OR REPLACE TABLE example.testdb.customer_orders AS
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna';
```

Comment:
You can use another timezone (if you want) - `Europe/Vienna` is not the standard for Trino's docs :)